gen-go-build.py

```diff
@@ -36,7 +39,7 @@ def maybe_gen(dir, go_version, arch):
         os.chmod(filename, 0755)
 
 def mac():
-    for mac_version in mac_versions:
+    for mac_version, go_versions in mac_versions.items():
```

Comment:
This shadows the global veriable name, and that's maybe not obvious. It would be better to use a different variable name in the loop, or all-caps the file-level constants.

Reply:
Yeah, I agree with this, will do.